offline_install_scripts/build_offlinebundle_dockerized.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Usage: invoked my "make offlinebundle_dockerized"
+# it is meant to run inside a docker container where it sets up the environment
+# it then calls then runs build_offlinebundle.sh as user
+apt-get update
+apt-get install -y zip unzip -q
```

Comment:
That would make sense if we were building an image, but we are only building a container. The contain is run with `--rm` so it gets wiped out at the end.